autoware_launch/config/planning/mission_planning/mission_planner/mission_planner.param.yaml

```diff
@@ -13,3 +13,4 @@
     consider_no_drivable_lanes: false # This flag is for considering no_drivable_lanes in planning or not.
     check_footprint_inside_lanes: true
     allow_reroute_in_autonomous_mode: true
+    goal_lanelet_transparency: 0.05
```

Comment:
It might be good to add 1 liner comment which describe what it is and the effect if we increase or decrease the value.